api.go

```diff
@@ -53,6 +55,8 @@ func GetLogger(ctx context.Context) Logger {
 // IdentityToken provides an ntoken for Athenz access for the authorization handler itself.
 type IdentityToken func() (string, error)
 
+type AthenzX509 func() (*tls.Config, error)
```

Comment:
Would `IdentityTLSCert` or `IdentityClientTLSCert` be more appropriate?

Reply:
sounds good. just that it can get confusing with tls cert for webhook itself for https so I thought keeping x509 explicit might make more sense. thoughts?

Reply:
yep, but let's add `Identity` prefix to be consistent with `IdentityToken` as they are used for the same purpse